submissions/Lu-sure/Popup/style.css

```diff
@@ -0,0 +1,207 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: content-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-size: 16px;
+}
+
+.navigation {
+  display: flex;
+  justify-content: flex-end;
+  height: 65px;
+  align-items: center;
+}
+
+ul {
+  list-style: none;
+}
+
+.nav-icon {
+  display: flex;
+  align-items: center;
+}
+
+.nav-text-item {
+  text-decoration: none;
+  color: #5f5f5f;
+  margin-right: 25px;
+}
+
+.nav-text-item:hover, .popup-bottom:hover {
+  text-decoration: underline;
+}
+
+.nav-img-item, .menu-img {
+  height: 32px;
+  width: 32px;
+}
+
+.nav-img-item {
+  margin-right: 25px;
+}
+
+.menu-item-border {
+  border: 3px solid transparent;
+  border-radius: 8%;
+}
+
+.menu-item-border:hover {
+  border-color: rgb(145, 181, 248);
+}
+
+.bell-item {
+  width: 26px;
+  height: 30px;
+}
+
+.bell-img {
+  width: 100%;
+  height: 100%;
+}
+
+.round-img {
+  border-radius: 50%;
+}
+
+#hoogle-menu, #more-button {
```

Comment:
Avoid naming by id and tag name, better to use the class name

Reply:
Ok. Did you mean avoid using id and tag name for css styles only?
 But I use a label for checkbox, it needs an ID, as I understand.
<input type="checkbox" id="hoogle-menu">
            <label for="hoogle-menu"> ...
<input type="checkbox" id="more-button">
                <label for="more-button" ...
is it ok to use IDs in this case? I needed these styles only for these two elements, it was so convinient.

Reply:
Only for CSS)